src/Components/NavLinks/index.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react"
+import { Link } from "react-router-dom"
+
+function NavLinks() {
+    const [modalClose, setModalClose] = useState (true)
+
+   function handleClickLink() {
+    setModalClose(false)
+   }
```

Comment:
You're kinda doubling up on logic here - almost the exact same code already exists within the Nav component. 

Rather than duplicating and tweaking it here, you should instead pass the handleClick function (from Nav) into this component as a prop which you can then put into your onClick events instead. 

That way your code is DRY-er and simpler, and whether or not the menu is shown is decided entirely by the Nav component.